fortinet_fortigate/plugin.spec.yaml

```
@@ -637,18 +637,33 @@ actions:
         type: string
         required: true
         example: InsightConnect Block Policy
+        order: 1
       address:
         title: Address
-        description: The address object name, IP address, CIDR IP address, or domain to check for
+        description: The Address Object name to check. If Enable Search is set to true then we search the addresses (IP, CIDR, doman) within the address object instead of matching the name
         type: string
+        example: 198.51.100.100
         required: true
-        example: MaliciousHost
+        order: 2
+      enable_search:
+        title: Enable Search
+        description: When enabled, the Address input will accept a IP, CIDR, or domain name to search across the available Address Objects in the system. This is useful when you don't know the Address Object by its name
```

Fix the typo in `doman`. We just fixed it in Palo

```suggestion
        description: When enabled, the Address input will accept a IP, CIDR, or domain name to search across the available Address Objects in the system. This is useful when you don't know the Address Object by its name
```